openstack/main.tf

```diff
@@ -44,7 +44,9 @@ variable master_count {
   default = 1
 }
 
-variable master_flavor {}
+variable master_flavor {
```

Comment:
This is modifications that doesn't really have to do with plugin, but a step to make script more generic, this way master nodes count can be 0 and also 0 node being default option